drakcore/drakcore/app.py

```diff
@@ -107,6 +107,13 @@ def logs(task_uid, log_type):
         return send_file(f.name, mimetype='text/plain')
 
 
+@app.route("/dumps/<task_uid>")
+def dumps(task_uid):
+    with NamedTemporaryFile() as f:
+        minio.fget_object("drakrun", task_uid + "/" + "dumps.zip", f.name)
+        return send_file(f.name, mimetype='text/plain')
```

Comment:
I'm unsure whether this would always work. I've seen some pattern where NamedTemporaryFile is manually destroyed in `after_request` or whatever the callback name was.

It strongly depends on the implementation of `send_file`, I belive.

Reply:
Well, I copied it from /logs endpoint. From my understanding it should work becase under the hood, `send_file` opens the file and stores the handle. So after returning and exiting the TempFile context, file is still referenced.

Reply:
I'm not sure if it's valid but i guess it works :thinking:

Reply:
[This SO answer](https://stackoverflow.com/a/33818894/13212650) seems to do exactly the same and it appears that using `NamedTemporaryFile` is perfectly fine in this case.

Reply:
Thanks @BonusPlay !